doc/source/gotchas.rst

```diff
@@ -242,6 +242,54 @@ Label-based slicing conventions
 Non-monotonic indexes require exact matches
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
+If the index of a ``DataFrame`` is monotonically increasing or decreasing, then the bounds
+of a label-based slice can be outside the range of the index, much like slice indexing a 
+normal Python ``list``.
+
+.. code-block:: python
+
+   >>> df_monotonic_index = pd.DataFrame(index=[2,3,3,4,5], columns=['data'], data=range(5))
+    
+   # no rows 0 or 1, but still returns rows 2, 3 (both of them), and 4:
+   >>> df_monotonic_index.loc[0:4, :]
```

Comment:
In this case it makes sense to use code blocks I think, as below there are some error messages shown (and they don't need to be shown fully). 
 But, maybe use IPython prompts for uniformity with the rest of the document?